py/torch_tensorrt/dynamo/conversion/impl/slice/ops.py

```diff
@@ -225,3 +225,36 @@ def tile(
     layer.mode = trt.SampleMode.WRAP
     set_layer_name(layer, target, name)
     return layer.get_output(0)
+
+
+def flip(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    dims: Sequence[int],
+) -> TRTTensor:
+    start_slice = []
+    output_shape = list(input.shape)
+    stride_slice = []
+
+    shape = input.shape
+    rank = len(shape)
+
+    for i in range(rank):
+        if i in dims:
```

Comment:
`dims` may need to be casted to positive dimension, if the user specifies a negative dim